### PR TITLE
CTDA-517: Update Test Support API to support IE028

### PR DESCRIPTION
**Pull request by jhkldev**

### CODE REVIEW

File: app/utils/Messages.scala [hunk 1]
```diff
@@ -204,4 +206,30 @@ object Messages {
 
     xml
   }
+
+  def generateIE028Message(): NodeSeq = {
+    val xml =
+      <CC028A>
+        <SynIdeMES1>{Strings.alpha(4)}</SynIdeMES1>
+        <SynVerNumMES2>{Strings.numeric(1)}</SynVerNumMES2>
+        <MesSenMES3>{Strings.alphanumeric(1, 35)}</MesSenMES3>
```
Comment (RobertLaverick) [inline app/utils/Messages.scala:215]: Can you create a sub task on CTDA-390 to remove all uses of MesSenMES3 (where relevant) in the test support API?

File: test/controllers/DepartureTestMessagesControllerSpec.scala [hunk 2]
```diff
@@ -121,7 +121,7 @@ class DepartureTestMessagesControllerSpec extends SpecBase with ScalaCheckProper
       val invalidRequest = Json.parse(
         """{
           |     "message": {
-          |         "messageType": "IE028"
+          |         "messageType": "IE005"
```
Comment (RobertLaverick) [inline test/controllers/DepartureTestMessagesControllerSpec.scala:124]: Is IE028 not good enough any more?

File: test/controllers/actions/ValidateMessageActionSpec.scala [hunk 3]
```diff
@@ -306,7 +353,7 @@ class ValidateMessageTypeActionSpec
       val exampleRequest: JsValue = Json.parse(
         """{
           |     "message": {
-          |         "messageType": "IE028"
+          |         "messageType": "IE005"
```
Comment (RobertLaverick) [inline test/controllers/actions/ValidateMessageActionSpec.scala:356]: Is IE028 not good enough?

File: app/utils/Messages.scala (hunk 1 above)
Comment (jhkldev) [reply app/utils/Messages.scala:215]: Done 👍

File: test/controllers/DepartureTestMessagesControllerSpec.scala (hunk 2 above)
Comment (jhkldev) [reply test/controllers/DepartureTestMessagesControllerSpec.scala:124]: IE028 is now considered a valid message 😄

File: test/controllers/actions/ValidateMessageActionSpec.scala (hunk 3 above)
Comment (jhkldev) [reply test/controllers/actions/ValidateMessageActionSpec.scala:356]: IE028 is now considered a valid message 😄

**Pull request merged**